docs/gff3_to_fasta.md

```diff
@@ -15,8 +15,8 @@ Extract sequences from specific regions of genome based on gff file.
     - **`pep`**: Translated peptide sequences (translation based on cds regions) for each record in the [`FASTA`] output. CDS features need to be included in the gff file.
     - **`user_defined`**: Specify parent and child features for fasta extraction via the -u argument, format [parent feature type] [child feature type].(e.g. `-st user_defined -u miRNA exon`)
 * **`translator` method for universal translation**: The `translator` method is feasible for
-    - translation from 64 combitions of [standard codons](https://en.wikipedia.org/wiki/DNA_codon_table) (Only standard codons and universal stop condons are considered.)
```

Comment:
It seems to me that this should not be changed back to broken links.

Reply:
Yeah, I'm not sure what happened there - https://en.wikipedia.org/wiki/DNA_codon_table apparently broke travis a couple of weeks ago, but now it's working again. @ShangYuChiang can you revert this link to https://en.wikipedia.org/wiki/DNA_codon_table? Sorry about the hassle. It looks like the web archive link doesn't work anymore.

Reply:
Ah, never mind. The new link does work. Let's leave this for now.